.editorconfig

```diff
@@ -0,0 +1,9 @@
+# top-most EditorConfig file
+root = true
+
+[*.{cpp,hpp}]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = tab
+tab_witdh = 8
```

Comment:
Is `.editorconfig` the kind of standard for defining code styles? Could be very useful to have one, outside of Emacs, but this looks like a good start. `tab_width` should be `4` though.

Reply:
It is indeed and has been adopted by many editors. See https://editorconfig.org/

I just noted that besides the incorrect tab width, I misspelled `width` as I always do ...